components/WorkSectionAsGallery.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import { ContentfulMedia, SimpleFragment } from 'models';
+
+import { Markdown } from 'components/base';
+import Work from 'components/Work';
+
+import get from 'utils/get';
+
+const renderWorkGalleryAssetBlock = (workGalleryBlock) => {
```

Comment:
it might be more readable and more React-like to make this a JSX component, then you can call it like a component, i.e. `<WorkGalleryAssetBlock />` and `<WorkGalleryTextBlock />` with the props

Reply:
shore thing!